checks/handlerCheck.go

```diff
@@ -0,0 +1,96 @@
+package checks
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/newrelic/newrelic-lambda-extension/config"
+	"github.com/newrelic/newrelic-lambda-extension/lambda/extension/api"
+)
+
+type handlerRuntime func(handlerConfigs) error
+
+type handlerConfigs struct {
+	handlerName string
+	conf        *config.Configuration
+}
+
+var handlerCheck = map[string]handlerRuntime{
+	"node":   node,
+	"python": python,
+}
+
+var pathExists = func(p string) error {
+	if _, err := os.Stat(p); os.IsNotExist(err) {
+		return err
+	}
+	return nil
+}
+
+var checkAndReturnRuntime = func() handlerRuntime {
+	for k, v := range handlerCheck {
+		p := fmt.Sprintf("/var/lang/bin/%s", k)
+		err := pathExists(p)
+		if err == nil {
+			return v
+		}
+	}
+	// If we make it here that means the runtime is
+	// custom and we don't want to throw an error
+	return nil
+}
+
+var handlerPath = "/var/task"
+
+func checkHandler(conf *config.Configuration, reg *api.RegistrationResponse) error {
+	r := checkAndReturnRuntime()
+	if r != nil {
+		h := handlerConfigs{
+			handlerName: reg.Handler,
+			conf:        conf,
+		}
+		err := r(h)
+		if err != nil {
+			return fmt.Errorf("Lambda handler is set incorrectly: %s", err)
```

Comment:
Let's be more explicit about what wasn't correct here. This is for the end user to solve their problem. If I'm a new customer and I screw this up, something like "Missing handler file customerHandler.js (NEW_RELIC_LAMBDA_HANDLER=customHandler.handler)" gives me a better idea of what is wrong.

I also think we should at least warn when the user isn't using the layer wrapper. That's a common mistake. Remember: these checks aren't fatal, just advisory.